app/views/users/home.html.erb

```diff
@@ -30,6 +30,9 @@
         <li>status is active: <%= user_org.active %></li>
         <% if user_org.admin? %>
           <li><button><%= link_to "Invite ppl", new_org_path(organization_id: user_org.organization_id) %></button></li>
+          <% if Organization.find(user_org.organization_id).stripe_id.blank? %>
```

Comment:
It's a best practice to not do any type of active record (sql or generally complex) queries in the view. If the `UserOrganization` has `belongs_to :organization`, then you could just do `user_org.organization.stripe_id.blank?`